prow/tide/tide.go

```diff
@@ -278,14 +275,16 @@ func (c *Controller) Sync() error {
 	}()
 	defer c.changedFiles.prune()
 
-	ctx := context.Background()
 	c.logger.Debug("Building tide pool.")
 	prs := make(map[string]PullRequest)
 	for _, q := range c.config().Tide.Queries {
-		results, err := newSearchExecutor(ctx, c.ghc, c.logger, q.Query()).search()
-		if err != nil {
+		results, err := search(c.ghc.Query, c.logger, q.Query(), time.Time{}, time.Now())
```

Comment:
searching between `time.Time{}` and `time.Now()` matches the previous `search()` behavior.